cli/script/command-parser.ts

```diff
@@ -41,10 +42,23 @@ function updateCheck(): void {
 
 function accessKeyAdd(commandName: string, yargs: yargs.Argv): void {
     isValidCommand = true;
-    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " <description>")
-        .demand(/*count*/ 3, /*max*/ 3)  // Require exactly two non-option arguments.
-        .example("access-key " + commandName + " \"VSO Integration\"", "Creates a new access key with the description \"VSO Integration\"");
+    yargs.usage(USAGE_PREFIX + " access-key " + commandName + " <accessKeyName>")
+        .demand(/*count*/ 3, /*max*/ 3)  // Require exactly three non-option arguments.
+        .example("access-key " + commandName + " \"VSTS Integration\"", "Creates a new access key with the name \"VSO Integration\" which expires by default in 60 days")
+        .example("access-key " + commandName + " \"One time key\" --maxAge 5m", "Creates a new access key with the name \"One time key\" which expires in 5 minutes")
+        .option("maxAge", { default: null, demand: false, description: "A duration string specifying the time for which the access key remains valid for use", type: "string" });
+
```

Comment:
I can appreciate why you went with "maxAge" as the name of this, but I can't help but feel like it sounds odd. The use of "max" is pretty strange, since I'm assuming our server has no intention of expiring the key sooner than the specified durarion, unlike how a browser or intermediary server might do with an HTTP response. I think as long as our docs and inline help description explain that this value is relative to the current time, as opposed to being an absolute date, we could call it "expiration" or "ttl" or whatever, and it wouldn't be confusing.